packages/core/database/lib/schema/diff.js

```diff
@@ -365,9 +365,8 @@ module.exports = (db) => {
         const dependencies = persistedTables
           .filter((table) => {
             const dependsOn = table?.dependsOn;
-            if (_.isNil(dependsOn)) return;
-            // FIXME: The array parse should not be necessary
-            return _.toArray(dependsOn).some((table) => table.name === srcTable.name);
+            if (_.isNil(dependsOn) || !_.isArray(dependsOn)) return;
```

Comment:
```suggestion
            if (!_.isArray(dependsOn)) return;
```

null or undefined should return false 👍

Reply:
good catch 😄